components/notifications/NotificationFeed.tsx

```
@@ -0,0 +1,226 @@
+										<div className="flex-1 min-w-0 space-y-1">
+											<div className="flex items-start justify-between gap-2">
+												<p className="text-sm font-medium text-foreground leading-tight">
+													{sender.name}
+												</p>
+												<p className="text-xs text-muted-foreground shrink-0">
+													{formatDistanceToNow(new Date(notification.inserted_at), {addSuffix: true})}
+												</p>
+											</div>
+											<div
+												className="text-sm text-muted-foreground leading-relaxed prose prose-sm max-w-none [&>*]:my-0"
+												dangerouslySetInnerHTML={{__html: body}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)